package.json

```diff
@@ -81,7 +81,7 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js && node versioned-build.js",
-    "test": "node scripts/test.js --no-watch --testPathIgnorePatterns=src/tests/accessibility",
+    "test": "node scripts/test.js --watch --testPathIgnorePatterns=src/tests/accessibility",
```

Comment:
why did you change this part?
won't it look for the accessibility test, which is a separate job?

Reply:
the `--watch` flag makes the tests watch for changes as you work. So if you change a file, as long as you had ran `npm run test` you would see the results of the tests immediately without running it again. with the `--no-watch` you need to run the tests command again

The `--testPathIgnorePatterns` flag is a different parameter that helps the runner know which file to ignore